build/download-manifests/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/projectsyn/steward/pkg/images"
+)
+
+var crds = []string{
+	"https://raw.githubusercontent.com/argoproj/argo-cd/{{VERSION}}/manifests/crds/application-crd.yaml",
+	"https://raw.githubusercontent.com/argoproj/argo-cd/{{VERSION}}/manifests/crds/appproject-crd.yaml",
+}
+
+func main() {
+	path, err := os.Getwd()
+	if err != nil {
+		panic(err)
```

Comment:
In general I would avoid panics, even for such a script. Printing the error and exiting is IMO just a lot more readable if the script fails.